src/backend/base/langflow/components/tools/DuckDuckGoSearchRun.py

```diff
@@ -0,0 +1,50 @@
+from langflow.base.io.text import TextComponent
+from langflow.io import StrInput, IntInput, Output
+from langflow.schema.message import Message
+from langchain_community.tools import DuckDuckGoSearchRun
+
+
+class DuckDuckGoSearchComponent(TextComponent):
```

Comment:
This is a tool, so you should build it as such.
```suggestion
from langflow.base.langchain_utilities.model import LCToolComponent
from langflow.io import StrInput, IntInput, Output
from langflow.schema.message import Message
from langchain_community.tools import DuckDuckGoSearchRun


class DuckDuckGoSearchComponent(LCToolComponent):
```